src/clims/plugins/demo/dnaseq/workflows/sequence.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+
+# TODO: Should we have a namespace from which plugin developers should import
+# (e.g. only clims.plugins)?
+
+from clims.services.workflow.camunda import CamundaWorkflow
+from clims.services import TextField
+
+SAMPLE_TYPES = ['DNA', 'RNA']
+PREP_METHODS = ['microwave', 'mixer']
+SEQUENCERS = ['iPhone', 'Android', 'Bang & Olufsen']
+
+
+class SequenceSimple(CamundaWorkflow):
+    """
+    Sequence a sample.
+
+    Note that since this is a `CamundaWorkflow`, the flow of the workflow is described in
+    a file with the same name as the class but the .bpmn extension. This class acts as glue between
+    the visual workflow definition and the data entry views that are required for completing the
+    workflow.
+    """
```

Comment:
Its very nice to see this. It really shows how it is all coming together beautifully in the user-facing APIs. Splendid!